Homeworks/Homework W8/SQLQuery_HomeworkW8.sql

```diff
@@ -0,0 +1,340 @@
+CREATE DATABASE ZooManagemnt;
+
+USE ZooManagemnt;
+-- Creating tables
+CREATE TABLE ZoneArea
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Zone VARCHAR(50)
+);
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(50),
+Zone INT REFERENCES ZoneArea(Id)
+);
+
+CREATE TABLE Animals
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species INT REFERENCES Species(Id)
+);
+
+CREATE TABLE Workers
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+FirstName VARCHAR (20),
+LastName VARCHAR (20),
+WorkArea INT REFERENCES ZoneArea(Id)
```

Comment:
ZoneAreaId (explanation in a comment below) + if the table is called ZoneArea the foreign key name should be similar so that the code does not become confusing.